tasks/freeradius.yml

```diff
@@ -152,3 +151,12 @@
     owner: freerad
     group: freerad
   notify: restart freeradius
+
+- name: Inner tunnel
+  template:
+    src: freeradius/openwisp_site.j2
```

Comment:
@atb00ker isn't this wrong?

Reply:
Yes, this was one of the causes of build failure,fixed it in the latest commit! :smile: